static/app/views/performance/transactionSummary/transactionOverview/content.tsx

```diff
@@ -213,6 +217,10 @@ function SummaryContent({
     t('timestamp'),
   ];
 
+  if (organization.features.includes('session-replay-ui')) {
+    transactionsListTitles.push(t('replayid'));
```

Comment:
```suggestion
    transactionsListTitles.push(t('replay id'));
```

add space to match `event id` and `trace id` column headers.